frontend/src/components/ProjectCardGrid/index.tsx

```diff
@@ -13,7 +13,7 @@ const ProjectCardGrid = ({
   intersectionObserverRef,
 }: ProjectCardGridProps) => {
   // Set the marker to load more projects to 6 cards before the end of the grid.
-  const loadMoreProjectsMarker = projects.length - 6
+  const loadMoreProjectsMarker = projects?.length - 6
```

Comment:
Will `loadMoreProjectsMarker` be `NaN` or `-6` when projects is `undefined`? How about when it's `null`?

Seems like a situation where it should be an empty array if there's no projects and never `null` or `undefined`.

Reply:
Resolved in 098c1853.